task_15/test/specs/example.e2e.js

```diff
@@ -0,0 +1,15 @@
+const LoginPage = require('../pageobjects/login.page');
+const SecurePage = require('../pageobjects/secure.page');
+
+describe('My Login application', () => {
+    it('should login with valid credentials', async () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887246)
```suggestion
    it("should login with valid credentials", async () => {
```